srcpkgs/libcap/template

```diff
@@ -56,11 +56,13 @@ libcap-progs_package() {
 	}
 }
 
-libcap-pam_package() {
-	conf_files="/etc/security/capability.conf"
-	short_desc+=" - PAM module"
-	pkg_install() {
-		vinstall pam_cap/capability.conf 644 etc/security
-		vmove usr/lib/security
+if [ "$CHROOT_READY" ]; then
```

Comment:
You should be using  `subpackages="..."` and `subpackages+=" ..."` instead of this.